### ISSUE TITLE
Analog input not supported

### ISSUE THREAD

**Issue by DrewUniverse** [Legend-of-Dragoon-Modding/Legend-of-Dragoon-Java#181]
I'm aware the controller config is an early hack atm; just noting this so we don't forget.

**Comment by Ink230** [comment]
PR #302 goes into the specifics of this issue.

The TL:DR is that 
1. The original input code was not compiled fully / not fully understood how it was achieved in retail
2. The current means of achieving auto-running as a config or as a default input is by a direct memory address flag which could provide conflicts with the 0x40 input flag, if it is always active. It is unproven if that 0x40 flag has any validation or checks to only be checked in SMap instances.

Investigation into the original input code or investigating how the script engine is handling 0x40 for running (as no run code for SMap.java is in the Java code) is needed to continue with the feature.

**Comment by LordMonoxide** [comment]
The original title/description was misleading, analog input wasn't handled at all.

In retail, you could:
- walk at any angle rather than being locked to 45 degrees
- run by tilting the thumbstick past the 50% threshold
